src/CMakeLists.txt

```
@@ -409,42 +412,40 @@ add_executable(openshot-player Qt/demo/main.cpp)
 target_link_libraries(openshot-player openshot)
 
 ############### TEST BLACKMAGIC CAPTURE APP ################
-IF (BLACKMAGIC_FOUND)
+if (BLACKMAGIC_FOUND)
 	# Create test executable
 	add_executable(openshot-blackmagic
 			examples/ExampleBlackmagic.cpp)
 
 	# Link test executable to the new library
 	target_link_libraries(openshot-blackmagic openshot)
-ENDIF (BLACKMAGIC_FOUND)
+endif()
 
 ############### INCLUDE SWIG BINDINGS ################
 add_subdirectory(bindings)
 
 ############### INSTALL HEADERS & LIBRARY ################
-set(LIB_INSTALL_DIR lib${LIB_SUFFIX}) # determine correct lib folder
 
 # Install primary library
-INSTALL(TARGETS openshot
-		ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
-		LIBRARY DESTINATION ${LIB_INSTALL_DIR}
-		RUNTIME DESTINATION ${LIB_INSTALL_DIR}
-		COMPONENT library )
+install(TARGETS openshot
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```

I accidentally included a change here to install the library DLLs into `bin'`, instead of into `lib/`, on Windows. Which is something I do want to do (as it's more correct when building Windows code), but we're not ready for that yet.

```suggestion
  RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR})
```